core/src/main/java/cucumber/runtime/Runtime.java

```diff
@@ -121,6 +121,12 @@ public void runFeature(CucumberFeature feature) {
         }
     }
 
+    public void runPickle(PickleEvent pickle) {
```

Comment:
There should be no `runPickle` on the `Runtime`, instead the `TestNGCucumberRunner` should do what for instance the [Android executor](https://github.com/cucumber/cucumber-jvm/blob/master/android/src/main/java/cucumber/runtime/android/CucumberExecutor.java#L113) and call `runPickle` on the `Runner`.